### PR TITLE
`hs sandbox create` duplicate name error handling

### PR DESCRIPTION
**Pull request by adamawang**
## Description and Context

Issue to track: https://git.hubteam.com/HubSpot/hubspot-cli-issues/issues/405

Updates "error" message when a duplicate name is detected to be more clear about where the duplicate name was found. I've also used a logger.warn to make it clearer, and updated the default account name to append the newly created sandbox hub ID to avoid suggesting yet another duplicate name. 

## Screenshots

Before:
image

After: 
Screenshot 2023-06-12 at 15 27 13

### CODE REVIEW

File: packages/cli/lib/sandboxes.js [hunk 1]
```diff
@@ -256,19 +256,20 @@ const saveSandboxToConfig = async (env, result, force = false) => {
...
         );
         const { name: promptName } = await enterAccountNamePrompt(
-          nameForConfig
+          nameForConfig + `_${result.sandbox.sandboxHubId}`
         );
         validName = promptName;
       } else {
         // Basic invalid name handling when force flag is passed
-        validName = nameForConfig + `_${Date.now()}`;
+        validName = nameForConfig + `_${result.sandbox.sandboxHubId}`;
```
Comment (adamawang) [inline packages/cli/lib/sandboxes.js:272]: Use the sandboxHubId instead of a date string for readability

File: packages/cli/lang/en.lyaml [hunk 2]
```diff
@@ -1077,7 +1077,7 @@ en:
             errors:
               invalidName: "You entered an invalid name. Please try again."
               nameRequired: "The name may not be blank. Please try again."
-              accountNameExists: "Account name \"{{ name }}\" already exists, please enter a different name."
+              accountNameExists: "Account with name \"{{ name }}\" already exists in the CLI config, please enter a different name."
```
Comment (adamawang) [inline packages/cli/lang/en.lyaml:1080]: Adjusted in two places to make it clearer what the "name" error is referencing

Comment (kemmerle) [reply packages/cli/lang/en.lyaml:1080]: Nice, this does make it much more clear 👍

File: packages/cli/lib/sandboxes.js (hunk 1 above)
Comment (kemmerle) [reply packages/cli/lib/sandboxes.js:272]: Agreed, this makes more sense to me, as well.

**Review by kemmerle — APPROVED** [review]
Reviewed the code and tested out locally, and these changes LGTM! 🚢

**Pull request merged**